frame/contracts/src/lib.rs

```diff
@@ -832,7 +844,7 @@ pub mod pallet {
 		},
 
 		/// Code with the specified hash has been stored.
-		CodeStored { code_hash: T::Hash },
+		CodeStored { code_hash: T::Hash, deposit_held: BalanceOf<T> },
```

Comment:
Maybe we should add who the uploader was. Then again you can look that up in storage. However, you can do the same for the deposit held. So we need to add `uploader` or remove `deposit_held`. Not sure what the best course of action is here.

Reply:
good point, similarly we could also add `remover` to the `CodeRemoved` event. WDYT?